src/utils/notif-handler.js

```diff
@@ -36,7 +37,7 @@ function getHandlerName(eventName) {
   return `${model}:${action}`;
 }
 
-function processHandlersFactory(handlers, userHandlerOptions = {}) {
+function processHandlersFactory(handlers, options = {}) {
```

Comment:
I think for deprecation period we should support both names. Otherwise it's just breaking.

Reply:
They are supported. Here it’s not a named argument so the internal name doesn’t matter no ?

Reply:
sorry, I've commented on wrong line, you are absolutely right. The breaking change is in smartNotifierHandler, let me comment it.